libs/core/src/components/pds-text/pds-text.tsx

```diff
@@ -25,6 +25,13 @@ export class PdsText {
   | 'success'
   | 'warning';
 
+  /**
+   * Sets the text decoration.
+   */
+  @Prop() decoration?:
+  | 'strikethrough'
```

Comment:
In CSS, strikethrough is line-through. Any preference on matching the naming? I was a bit on the fence here. Thoughts appreciated!

Reply:
I think from a customers perspective strike through is the term they'll be accustomed to.

Reply:
Agreed. Thanks @ju-Skinner!